Release Note: [BUGS-8984] Terminus 3.6.1 (#9336)
pantheon-systems/documentation: source/releasenotes/2024-12-04-terminus-361.md

---

> title: "Terminus 3.6.1 released"

Reviewer: ```suggestion
title: "Terminus 3.6.1 release now available"
```

published_date: "2024-12-04"
categories: [tools-apis, new-feature]
---

> Terminus [3.6.1](https://github.com/pantheon-systems/terminus/releases/tag/3.6.1) was released. It contains the latest bug fixes and improvements for this tool.

Reviewer: ```suggestion
Terminus [3.6.1](https://github.com/pantheon-systems/terminus/releases/tag/3.6.1) is now available as of November 30, 2024. This minor update contains the latest bug fixes and improvements for this tool.
```

> You can get the full details about this version in the release page linked above.

Reviewer: ```suggestion
For more information about this release, visit the [GitHub release page](https://github.com/pantheon-systems/terminus/releases/tag/3.6.1).
```
